Gordon360/Services/ProfileService.cs

```diff
@@ -359,6 +368,50 @@ public async Task UpdateImagePrivacyAsync(string username, string value)
 
             return profile.ToObject<ProfileViewModel>();
         }
+        public void SendUpdateEmail(string username, string email_content)
+        {
+            using (var smtp = new SmtpClient())
+            {
+                var account = _accountService.GetAccountByUsername(username);
+                //string to_email = "devrequest@gordon.edu";
+                /*
+                 * TO EMAIL TO BE CHANGED TO DEVREQUEST (commented above) change to_email variable for testing
+                 */
+                string bcc_email =  _context.ACCOUNT.FirstOrDefault(x => x.gordon_id == account.GordonID)?.email ?? ""; 
+                string to_email = "devrequest_placeholder@gordon.edu";
+                string from_email = serviceEmail.Username;
+                string subject = String.Format("ALUMNI INFORMATION UPDATE REQUEST FOR ID: {0}", account.GordonID);
```

Comment:
Perhaps this subject shouldn't be so volatile...
```suggestion
                string subject = String.Format("Alumni Information Update Request for ID: {0}", account.GordonID);
```

Reply:
Sound good, I think that was just what I got from the previous implementations